.rubocop.yml

```diff
@@ -2,5 +2,6 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
+inherit_from: .rubocop_todo.yml
```

Comment:
This will vanish on next modulesync update.

Reply:
Looking at https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.rubocop.yml.erb#L5

```erb
<% if File.exist?(File.join(@metadata[:workdir], '.rubocop_todo.yml')) -%>
inherit_from: .rubocop_todo.yml
<% end -%>
```

I think not:

Reply:
w00t! :heart_eyes: